docsrc/commands.rst

```diff
@@ -140,3 +140,45 @@ This can be changed using ``require_command`` property.
    local parser = argparse()
       :require_command(false)
    parser:command "install"
+
+Command summaries
+-----------------
+
+The description for commands shown in the main parser help message can be set
+with the ``summary`` property.
+
+.. code-block:: lua
+   :linenos:
+
+   parser:command "install"
+      :summary "Install a rock."
+      :description "Install a rock.\n\nA longer description."
```

Comment:
I think I'd prefer a description here that doesn't start with the summary verbatim.